Formula/libxlsxwriter.rb

```diff
@@ -0,0 +1,35 @@
+class Libxlsxwriter < Formula
+  desc "Libxlsxwriter: A C library for creating Excel XLSX files."
+  homepage "https://libxlsxwriter.github.io/"
+  url "https://github.com/jmcnamara/libxlsxwriter/archive/RELEASE_0.7.3.tar.gz"
+  sha256 "84dbbd164b440695119ded3a8e2b6cac07d0d284343ab977c88dc0418d8fcde5"
+
+  depends_on "lzlib"
+
+  def install
+    lib.mkdir
+    include.mkdir
+    system "make", "V=1"
+    system "make", "install", "INSTALL_DIR=#{prefix}", "V=1"
+  end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include "xlsxwriter.h"
+
+      int main() {
+          lxw_workbook  *workbook  = workbook_new("myexcel.xlsx");
+          lxw_worksheet *worksheet = workbook_add_worksheet(workbook, NULL);
+          int row = 0;
+          int col = 0;
+
+          worksheet_write_string(worksheet, row, col, "Hello me!", NULL);
+
+          return workbook_close(workbook);
+      }
+    EOS
+
+    system ENV.cc, "test.c", "-L#{lib}", "-I#{include}", "-lxlsxwriter", "-o", "test"
+    system "./test"
```

Comment:
Test is great! Please also add a final check that the file created actually exists, after running `test`